01_texinfo-usr-local.patch by James Troup <james@nocrew.org>

diff -urNad a/gdbm.texinfo b/gdbm.texinfo
--- a/gdbm.texinfo	2002-10-15 22:59:38.000000000 +0100
+++ b/gdbm.texinfo	2003-05-05 15:14:21.000000000 +0100
@@ -214,7 +214,7 @@
 int gdbm_fdesc(dbf);
 @end example
 
-The @code{gdbm.h} include file is often in the @file{/usr/local/include}
+The @code{gdbm.h} include file is often in the @file{/usr/include}
 directory. (The actual location of @code{gdbm.h} depends on your local
 installation of @code{gdbm}.)
 
@@ -712,7 +712,7 @@
 and want to use @code{gdbm} files, execute the following @code{cc} command:
 
 @example
-cc ... -L/usr/local/lib -lgdbm -lgdbm_compat
+cc ... -lgdbm -lgdbm_compat
 @end example
 
 @node Conversion, Bugs, Compatibility, Top
